test-suite/Makefile

```diff
@@ -27,7 +27,7 @@
 # If Coq is not configured, we can't run the makefile.
 ifneq ($(wildcard ../config/Makefile ../_build/default/config/Makefile),)
 -include ../config/Makefile
--include ../_build/default/config/Makefile
+-include ../_build/$(DUNE_CTX)/config/Makefile
```

Comment:
should be `default` when not set
also should be used in the ifneq

Reply:
Indeed, thanks; for now let's see if the upload and test-suite work, I leave this comment open to properly implement this.